src/operators.js

```diff
@@ -1,21 +1,9 @@
 import { filter } from 'rxjs/operators';
 
-const keyHasType = (type, key) => {
+const matchesType = type => key => {
   return type === key || typeof key === 'function' && type === key.toString();
 };
 
-export const ofType = (...keys) => (source) => source.pipe(
-  filter(({ type }) => {
-    const len = keys.length;
-    if (len === 1) {
-      return keyHasType(type, keys[0]);
-    } else {
-      for (let i = 0; i < len; i++) {
-        if (keyHasType(type, keys[i])) {
-          return true;
-        }
-      }
-    }
-    return false;
-  })
+export const ofType = (...keys) => source => source.pipe(
+  filter(({ type }) => keys.some(matchesType(type)))
```

Comment:
It could also be `keys.includes` but according to [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) that is not supported on all browsers. I'm not sure if IE must be supported.  Another option could be to use `keys.indexOf(...) !== -1` but that doesn't read like English where as `keys.some(matchesType(type)` does (sort of).